packages/vuetify/src/components/VTextField/VTextField.ts

```diff
@@ -120,9 +120,9 @@ export default baseMixins.extend<options>().extend({
       },
     },
     isDirty (): boolean {
-      return (this.lazyValue != null &&
-        this.lazyValue.toString().length > 0) ||
-        this.badInput
+      return this.lazyValue != null
+        ? this.lazyValue.toString().length > 0
+        : this.badInput
```

Comment:
What do you think about:
```ts
this.lazyValue?.toString().length > 0 || this.badInput
```

Reply:
Tried that, if it's nully it's like `undefined > 0 || this.badInput` which is a type error

Reply:
if `this.lazyValue != null` is TRUE
and `this.lazyValue.toString().length > 0` is FALSE
and `this.badInput` is TRUE

`(this.lazyValue != null && this.lazyValue.toString().length > 0) || this.badInput` === `(TRUE && FALSE) || TRUE` === `TRUE`
`this.lazyValue != null ? this.lazyValue.toString().length > 0 : this.badInput` === `TRUE ? FALSE : TRUE` === `FALSE`

Reply:
:facepalm: 

`undefined > 0` isn't an error any more so we can use `?.` here now